Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -383,7 +389,7 @@ var userLogOptions = {
 };
 ```
 
-These options should retrieve the last ten activities for the current user in descending order since the start of 2018, we pass the options into the **getPagedUserLog** like so:
+These options should retrieve the last ten activities for the current user in descending order since the start of 2018. We pass the options into the **getPagedUserLog** like so:
```

Comment:
```suggestion
These options should retrieve the last ten activities for the current user in descending order since the start of 2018. We pass the options into the `getPagedUserLog` like so:
```